test/hts-precompile/hrc/HRCContract.js

```diff
@@ -64,9 +64,6 @@ describe('HRC tests', function () {
     )
     // create a contract object for the token
     hrcToken = new Contract(tokenAddress, IHRC, signers[0])
-    console.log('hrcContract: ', hrcContract.address)
```

Comment:
Q: why did you remove this? Maybe revert

Reply:
Restored.

Reply:
Not restored, did you stage and not push it?